ui-admin/src/study/participants/KitRequests.tsx

```diff
@@ -74,19 +75,25 @@ export default function KitRequests({ enrollee, studyEnvContext, onUpdate }:
     getCoreRowModel: getCoreRowModel()
   })
 
-  return <div>
-    <h2 className="h4">Kit requests</h2>
-    {user?.superuser &&
-      <button className='btn btn-secondary' onClick={() => setShowRequestKitModal(true)}>
-        <FontAwesomeIcon icon={faPlus}/> Create a kit request
-      </button>
-    }
+  return <div className="card w-75 border shadow-sm mb-3">
+    <div className="card-header border-bottom bg-white d-flex flex-row align-items-center">
+      <div className="d-flex justify-content-between align-items-center w-100">
+        <div className="fw-bold lead my-1">Kit Requests</div>
+        {user?.superuser &&
+            <Button onClick={() => setShowRequestKitModal(true)}
+              variant="light" className="border m-1">
+              <FontAwesomeIcon icon={faPlus} className="fa-lg"/> Request a kit
+            </Button>
+        }
+      </div>
+    </div>
     {showRequestKitModal && <RequestKitModal
       studyEnvContext={studyEnvContext}
       enrolleeShortcode={enrollee.shortcode}
       onDismiss={() => setShowRequestKitModal(false)}
       onSubmit={onSubmit}
     />}
     {basicTableLayout(table)}
+    {<div className='my-3'>{renderEmptyMessage(enrollee.kitRequests, 'No kit requests')}</div>}
   </div>
```

Comment:
nit: should we reuse infocard for consistency? also second nit: margin at the bottom of the table
```suggestion
  return <InfoCard>
    <InfoCardHeader>
      <div className="d-flex justify-content-between align-items-center w-100">
        <div className="fw-bold lead my-1">Kit Requests</div>
        {user?.superuser &&
            <Button onClick={() => setShowRequestKitModal(true)}
              variant="light" className="border m-1">
              <FontAwesomeIcon icon={faPlus} className="fa-lg"/> Request a kit
            </Button>
        }
      </div>
    </InfoCardHeader>
    {showRequestKitModal && <RequestKitModal
      studyEnvContext={studyEnvContext}
      enrolleeShortcode={enrollee.shortcode}
      onDismiss={() => setShowRequestKitModal(false)}
      onSubmit={onSubmit}
    />}
    {basicTableLayout(table, { tableClass: 'table m-0' })}
    {enrollee.kitRequests.length === 0 && <div className='my-3'>
      {renderEmptyMessage(enrollee.kitRequests, 'No kit requests')}
    </div>}
  </InfoCard>
```

Reply:
that's a great call, updating these to use infocard